config/routes.rb

```diff
@@ -161,6 +161,7 @@
     namespace :profile do
       resource :alternate_phone, only: %i[show create]
       resource :email, only: %i[show create]
+      resource :email_addresses, only: %i[create update]
```

Comment:
Are we leaving the old email routes in place? It feels not ideal to have /v0/profile/email/... and /v0/profile/email_addresses/... living side-by-side

Reply:
Yes, we are.  We decided to keep them for the time being.  Those might be used in the future, and do not want to throw away that work.